quickstart/docker-compose.yml

```diff
@@ -98,6 +98,22 @@ services:
       - 9090
     volumes:
       - ../quickstart/exampledata/config/prometheus/prometheus.yml:/opt/bitnami/prometheus/conf/prometheus.yml
-
+  config:
+    image: nginx:latest
+    container_name: config
+    profiles:
+      - basic_auth
+    network_mode: host
+    expose:
+      - 8080
+    volumes:
+      - ../quickstart/exampledata:/usr/share/nginx/html:ro
+      - ../quickstart/exampledata/config/nginx/nginx.conf:/etc/nginx/nginx.conf:ro
+      - ../quickstart/exampledata/config/nginx/conf.d:/etc/nginx/conf.d:ro
+  keycloak:
+    image: bitnami/keycloak:latest
+    container_name: keycloak
+    profiles:
+      - oauth2
```

Comment:
Can you add this to the README.md? There is a table with the exposed ports where you could add these two new services. Also it could be worth it if you update the description on how to run logprep with the config that is provided by the nginx.

Reply:
done